AlphaWallet/Analytics/Models/AnalyticsEventPropertyValue.swift

```diff
@@ -3,23 +3,51 @@
 import Foundation
 
 protocol AnalyticsEventPropertyValue {
+    var value: Any { get }
```

Comment:
Can you try to replace the call to `convertParameterToSdkSpecificVersion(_:)` with `value`?